CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.1] -  2023-02-20
+
+### Changed
+
+- Changed the helm charts to version 2.0.8 to upload with default values on pre prod
```

Comment:
The CHANGELOG should describe the fixes of your bugs. So i would propose something like:
"Fix whitespace handling that removed apiVersion declaration from Helm templates"
PRE-PROD is irrelevant for your CHANGELOG